sentences/da/_common.yaml

```diff
@@ -278,18 +278,23 @@ expansion_rules:
   gardin: "gardin[(et|er|erne)]"
   i_på: "(i | på)"
   koldt_varmt: "(kold[t] | varm[t])"
+  koldt: "(kold([t]|[e])|kold)"
   luk: "luk [for]"
   lys: "lys[(ene|et)]"
   lysstyrke: "{brightness}[%| procent]"
-  navn: "{name}[(en|et|n|t)]"
+  navn: "([den]|[det]) {name}[(en|ene|et|n|t|men|ten)][s]"
```

Comment:
I added `([den]|[det])` here because I'd say things like:
> Er den anden garage dør lukket?

But maybe I should change it to

"([den]|[det] {name}|{name}[(en|ene|et|n|t|men|ten)][s]}"

So it matches "den anden garage dør",  "garage døren" but not "den anden garage døren".